python/lsst/ts/electrometer/controller.py

```diff
@@ -220,6 +220,9 @@ async def connect(self) -> None:
                 f"Expected:\n {expected} \n but got: \n{res} \n"
             )
             raise RuntimeError("Communication verification failed.")
+        await self.send_command(f"{self.commands.reset_device()}")
+        await self.send_command(f"{self.commands.get_measure(1)}", has_reply=True)
```

Comment:
@couger01 should comment here... but glancing at the code maybe that 1 should be an enum?

Reply:
Use ReadingOption.LATEST instead of 1.

Reply:
Ok. I took this straight from `stop_scan()`. Should I change it there also?

Reply:
Yes. but @couger01 should confirm.

Reply:
@couger01 How would I actually implement this? And do you want it elsewhere in the code that get_measure() is used?